app.rb

```diff
@@ -0,0 +1,112 @@
+require_relative 'person'
+require_relative 'student'
+require_relative 'teacher'
+require_relative 'book'
+require_relative 'rental'
+
+class App
+  attr_accessor :people, :books, :rentals
+
+  def initialize
+    @people = []
+    @books = []
+    @rentals = []
+  end
+
+  def list_people
+    puts 'List of People:'
+    @people.each { |person| puts "ID: #{person.id}, #{person.class.name}: #{person.name}" }
+  end
+
+  def list_books
+    puts 'List of Books:'
+    @books.each_with_index { |book, index| puts "#{index}. #{book.title} by #{book.author}" }
+  end
+
+  def create_person
+    puts "Enter the person's name:"
+    name = gets.chomp
+
+    puts "Enter the person's age:"
+    age = gets.chomp.to_i
+
+    puts 'Is the person a student? (yes/no):'
+    is_student = gets.chomp.downcase
+
+    if is_student == 'yes'
+      puts 'Does the student have parent permission? (yes/no):'
+      parent_permission = gets.chomp.downcase == 'yes'
+      @people << Student.new(name, age, parent_permission: parent_permission)
+      puts "#{name} has been added as a student."
+    end
```

Comment:
- [ ] @berylcl Great work implementing the change requested however am still not able to create a student I get the error below when I choose the parent permission (yes or no), it seems your student class only accepts 2 parameters on the initialize method.
![image](https://github.com/berylcl/OOP-school-library/assets/61408860/c63cbb09-da0d-4c5b-995f-4679a5f84f01)